Runtimes/Supplemental/cmake/modules/gyb.cmake

```diff
@@ -0,0 +1,47 @@
+find_package(Python3 REQUIRED COMPONENTS Interpreter)
```

Comment:
Is this a copy from the core libraries gyb implementation? It really is unfortunate that we do not have a `Runtimes/cmake/modules` to share some of these through.

@etcwilde - what do you think of creating that?

Reply:
Yep, it's a copy in the same way we have [AvailabilityMacros](https://github.com/swiftlang/swift/blob/main/Runtimes/Supplemental/cmake/modules/AvailabilityMacros.cmake)

Reply:
I've been doing my best to prevent cross-pollination between the layers.
Worried about folks trying to be overly DRY and ending up with another umbrella `add_swift_target_library` function with 97 arguments.